Source/NSAccessibilityCustomRotor.m

```diff
@@ -25,89 +25,131 @@
 #import "AppKit/NSAccessibilityCustomRotor.h"
 
 @implementation NSAccessibilityCustomRotor
-  
+
 - (instancetype) initWithLabel: (NSString *)label
             itemSearchDelegate: (id<NSAccessibilityCustomRotorItemSearchDelegate>)delegate
 {
-  return nil;
+  self = [super init];
+  if (self != nil)
+    {
+      _type = NSAccessibilityCustomRotorTypeCustom; // default when label initializer used
+      ASSIGNCOPY(_label, label);
+      _itemSearchDelegate = delegate; // delegates not retained in Cocoa typically (weak)
+    }
+  return self;
 }
 
 - (instancetype) initWithRotorType: (NSAccessibilityCustomRotorType)rotorType
                 itemSearchDelegate: (id<NSAccessibilityCustomRotorItemSearchDelegate>)delegate
 {
-  return nil;
+  self = [super init];
+  if (self != nil)
+    {
+      _type = rotorType;
+      _itemSearchDelegate = delegate;
+    }
+  return self;
 }
 
 - (NSAccessibilityCustomRotorType) type
 {
-  return 0;
+  return _type;
 }
 
 - (void) setType: (NSAccessibilityCustomRotorType)type
 {
+  _type = type;
 }
 
 - (NSString *) label
 {
-  return nil;
+  return _label;
 }
 
 - (void) setLabel: (NSString *)label
 {
+  ASSIGNCOPY(_label, label);
 }
 
 - (id<NSAccessibilityCustomRotorItemSearchDelegate>) itemSearchDelegate
 {
-  return nil;
+  return _itemSearchDelegate;
 }
 
 - (void) setItemSearchDelegate: (id<NSAccessibilityCustomRotorItemSearchDelegate>) delegate
 {
+  _itemSearchDelegate = delegate;
 }
 
 - (id<NSAccessibilityElementLoading>) itemLoadingDelegate
 {
-  return nil;
+  return _itemLoadingDelegate;
 }
 
 - (void) setItemLoadingDelegate: (id<NSAccessibilityElementLoading>) delegate
 {
+  _itemLoadingDelegate = delegate;
+}
+
+- (void) dealloc
+{
+  RELEASE(_label);
+  [super dealloc];
 }
-  
+
 @end
 
 // Results...
 @implementation NSAccessibilityCustomRotorItemResult : NSObject
 
 - (instancetype)initWithTargetElement:(id<NSAccessibilityElement>)targetElement
 {
-  return nil;
+  self = [super init];
+  if (self != nil)
+    {
+      _targetElement = targetElement;
+      _targetRange = NSMakeRange(0, 0);
```

Comment:
Did you verify this specific value? The old value `NSMakeRange(0,NSNotFound)` seems more plausible to me.